dataplicity/constants.py

```diff
@@ -16,3 +34,18 @@
 # TODO: Replace this with a sensible chunk size once we identify the
 # issue with ssh over Porthole
 CHUNK_SIZE = 1024 * 1024
+
+# Maximum number of services (port forward/commands/file etc)
+LIMIT_SERVICES = get_environ_int("DATAPLICITY_LIMIT_SERVICES", 500)
```

Comment:
I think an RPi would really struggle to get anywhere near 500 simultaneous requests. Note this isn't even requests per second, which I would expect to be at least double.

The exception would be websockets and other long-running connections, which would tie up a single service.